sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StandardMetricsExtractionProcessor.cs

```diff
@@ -5,15 +5,19 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Diagnostics.Metrics;
+using System.Runtime.Serialization;
```

Comment:
is this needed?
Looks like something VS might have added while trying too hard to "help" :)